etc/bin/make-tarball

```diff
@@ -1,3 +1,19 @@
+# THIS FILE IS PART OF THE CYLC SUITE ENGINE.
```

Comment:
The shebang needs to come first... see Travis shellcheck error:

![image](https://user-images.githubusercontent.com/304786/65941968-c39c5b00-e488-11e9-87ea-935119ca6a07.png)

Reply:
Indeed, I saw I did that mistake in the other files but missed fixing that one...